### ISSUE TITLE
Exception while calling fromZigbee converter: Value '2' is not allowed, expected one of [object Object],[object Object],[object Object]

### ISSUE THREAD

**Issue by martiperez** [Koenkk/zigbee-herdsman-converters#6755]
TL;DR: Are you sure the line in the link below shouldn't have been **2**: 'holiday'? (instead of the 3)
https://github.com/Koenkk/zigbee-herdsman-converters/blob/92a015f558be1bd5fad1c605c0e52f9ad4e1147b/lib/tuya.js#L959

------------------------------------------

I noticed this error happening after I programmed the [TuYa TV02-Zigbee](https://www.zigbee2mqtt.io/devices/TV02-Zigbee.html#tuya-tv02-zigbee) for "holiday mode". 
![image](https://github.com/Koenkk/zigbee-herdsman-converters/assets/29499415/2104e0fa-e9bd-4c5a-9784-b2370c191279)

After investigating on the topic, I noticed a couple of tickets already which pointed out to this issue (and its solution, thanks to @guandalf !):
- https://github.com/Koenkk/zigbee-herdsman-converters/pull/3185#pullrequestreview-1379210337
- https://github.com/Koenkk/zigbee2mqtt/issues/16648

The solution (already provided in the TL;DR) would be to change the 3 for a 2 in the line 959:
https://github.com/Koenkk/zigbee-herdsman-converters/blob/92a015f558be1bd5fad1c605c0e52f9ad4e1147b/lib/tuya.js#L956-L960

**Comment by Koenkk** [comment]
Done!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.
